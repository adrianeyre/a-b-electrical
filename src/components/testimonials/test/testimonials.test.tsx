import React from 'react';
import { shallow } from 'enzyme';

import Testimonials from '../testimonials';

describe('Testimonials', () => {
	it('Should render correctly', () => {
		const testimonialsComponent = shallow(<Testimonials />);

		expect(testimonialsComponent).toMatchSnapshot();
	});
});
