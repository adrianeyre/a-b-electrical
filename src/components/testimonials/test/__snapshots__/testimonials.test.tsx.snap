// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials Should render correctly 1`] = `ShallowWrapper {}`;
